test/s3-resumption/mzcompose.py

```diff
@@ -17,7 +17,10 @@
 
 SERVICES = [
     Localstack(),
-    Materialized(environment=["MZ_LOG_FILTER=dataflow::source::s3=trace,info"]),
+    Materialized(
+        environment=["MZ_LOG_FILTER=dataflow::source::s3=trace,info"],
+        options=["--unsafe-mode"],
+    ),
```

Comment:
Ah, this looks like it wants `environment_extra` rather than `environment`. Then you won't need to pass `--unsafe-mode` by hand!

```suggestion
    Materialized(
        environment_extra=["MZ_LOG_FILTER=dataflow::source::s3=trace,info"],
    ),
```